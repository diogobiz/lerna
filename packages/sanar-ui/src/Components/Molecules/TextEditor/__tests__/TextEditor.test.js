import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESTextEditor from '../'

it('renders correctly', () => {
    const component = <ESTextEditor />

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
