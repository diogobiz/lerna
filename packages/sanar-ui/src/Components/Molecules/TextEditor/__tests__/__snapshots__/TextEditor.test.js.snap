// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-text-editor"
  style={
    Object {
      "height": 356,
    }
  }
>
  <div
    className="ant-spin-nested-loading"
  >
    <div>
      <div
        className="ant-spin ant-spin-spinning es-spin es-spin__dark"
        style={Object {}}
      >
        <span
          className="ant-spin-dot ant-spin-dot-spin"
        >
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
        </span>
      </div>
    </div>
    <div
      className="ant-spin-container ant-spin-blur"
    >
      <div
        contentEditable="true"
      />
    </div>
  </div>
</div>
`;
